src/facts/leafcutter-bees.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Leafcutter Bees
+image:
+  src: ../images/leafcutter-bee.jpg
+  alt: A leafcutter bee cutting a leaf
+  creator: Bernhard Plank
+  license: CC BY-SA 3.0
+  url: https://en.wikipedia.org/wiki/Megachilidae#/media/File:Leafcutter_bee_by_Bernhard_plank.jpg
```

Comment:
This is not the canonical URL for this image. Please use the Wikimedia Commons URL.
```suggestion
  url: https://commons.wikimedia.org/wiki/File:Leafcutter_bee_by_Bernhard_plank.jpg
```